examples/typemap.php

```diff
@@ -0,0 +1,121 @@
+<?php
+declare(strict_types=1);
+
+namespace MongoDB\Examples;
+
+use MongoDB\BSON\ObjectId;
+use MongoDB\BSON\Unserializable;
+use MongoDB\Client;
+use UnexpectedValueException;
+
+use function dirname;
+use function getenv;
+use function is_array;
+use function var_dump;
+
+require dirname(__FILE__) . '/../vendor/autoload.php';
+
+class TypemapEntry implements Unserializable
```

Comment:
A minor nit, but perhaps you want to use `TypeMapEntry` and `TypeMapEmail` for consistency with `$typeMap`. I don't feel strongly either way, but just pointing it out in case you didn't give it any thought.

Reply:
I can never remember the correct spelling. Changed to TypeMap.